.github/workflows/NativeTests.yaml

```diff
@@ -63,9 +64,71 @@ jobs:
             --define maven.test.skip=true \
             --define maven.javadoc.skip=true \
             install
-      - name: Retry install on failure
-        id: install2
-        if: steps.install1.outcome == 'failure'
+      # Unlike for the other NativeTests job, we don't block here as these run in parallel.
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: '22.3.2'
+          java-version: '17'
```

Comment:
FYI - This github action `setup-graal` has some changes since we last introduced them following GraalVm version naming changes. [readme](https://github.com/graalvm/setup-graalvm#migrating-from-graalvm-223-or-earlier-to-the-new-graalvm-for-jdk-17-and-later). If we want to move to test against graalvm for jdk 17